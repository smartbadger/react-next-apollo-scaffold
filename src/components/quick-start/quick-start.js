import styled from "styled-components"

const Welcome = styled.div`
    display: flex;
    justify-content: center;
    align-items: center;
    width: 100%;
    height: 100vh;
`
const Button = styled.button`
    background-color: ${props => props.bgColor ? props.bgColor:props.theme.bgColor};
    border-radius: 5px;
    &:hover {
        ${props => props.bgColorHover ? props.bgColor:props.theme.bgColorHover}
    }
`
const Message = styled.h1`
    font-family: ${props => props.font ? props.font : props.theme.font} 
`
export {Welcome, Message, Button}