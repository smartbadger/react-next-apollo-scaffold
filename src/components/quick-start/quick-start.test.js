
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import {Button} from'./quick-start'

test('it works', () => {
  const tree = renderer.create(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
})