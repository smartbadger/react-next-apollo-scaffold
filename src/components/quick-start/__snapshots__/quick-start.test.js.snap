// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  border-radius: 5px;
}

<button
  className="c0"
/>
`;
